docker/docker-compose.yaml

```diff
@@ -17,7 +17,7 @@
 version: '3.8'
 services:
   streampark-console:
-    image: apache/streampark:latest
+    image: apache/streampark:2.1.4
```

Comment:
Why is there no latest? Does this mean that we need to modify the yaml file every time we release a version?